services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
+/* eslint-disable react/prop-types */
+import React, { createContext, useState } from 'react';
+import { useChessBoard } from '../hooks/useChessBoard';
+import { NUMBER_OF_ROWS, NUMBER_OF_COLUMNS } from '../constants/board';
+import '../static/style/gameScore.css';
+
+export const ChessBoardContext = createContext();
+let num;
+
+const ChessBoardContextProvider = ({ children }) => {
+  const [selectedItem, setSelectedItem] = useState('');
+  const [board, setBoard] = useChessBoard();
+
+  const placeItemOnBoard = (row, tile) => {
+    if (selectedItem) {
+      const copyOfBoard = [...board];
+      copyOfBoard[row][tile].items.push(
+        selectedItem,
+        calculateTheScore(row, tile, selectedItem),
+        // + calculateTheScore(row, tile, selectedItem),
+      );
+      setBoard(copyOfBoard);
+      setSelectedItem('');
+    }
+  };
+
+  const calculateTheScore = (row, tile, selectedItem) => {
+    var selectedX = 0;
+    var selectedY = 0;
+    var selectedPiece = 0;
+
+    const xValues = new Map(
+      Object.entries({
+        1: 1,
+        2: 2,
+        3: 3,
+        4: 5,
+        5: 8,
+        6: 13,
+      }),
+    );
+    const yValues = new Map(
+      Object.entries({
+        1: 1,
+        2: 2,
+        3: 3,
+        4: 5,
+        5: 8,
+        6: 13,
+      }),
+    );
+
+    const pieces = new Map(
+      Object.entries({
+        pawn: 1,
+        knight: 2,
+        bishop: 3,
+        rook: 5,
+        king: 8,
+        queen: 13,
+      }),
+    );
+
+    for (const [key, value] of xValues.entries()) {
+      if ((NUMBER_OF_ROWS - row).toString() === key) {
+        selectedX = value;
+      }
+    }
+
+    for (const [key, value] of yValues.entries()) {
+      if (tile.toString() === key) {
+        selectedY = value;
+      }
+    }
+
+    for (const [key, value] of pieces.entries()) {
+      if (selectedItem === key) {
+        selectedPiece = value;
+      }
+    }
+
+    num = selectedX + selectedY + selectedPiece;
+  };
+
+  return (
+    <>
+      <ChessBoardContext.Provider
+        value={{
+          setSelectedItem,
+          selectedItem,
+          placeItemOnBoard,
+          board,
+        }}
+      >
+        {children}
+      </ChessBoardContext.Provider>
+      <div>
+        <label className="id">Game score: {num}</label>
```

Comment:
`id` is a bad name for css class in this case. Please pick something that describes this label from business logic point of view. e.g. `game-score`